imx8mm-var-dart.coffee

```diff
@@ -0,0 +1,54 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+ 
+IMX8M_VAR_DART_FLASH = 'Set the SW7 BOOT SELECT switch to SDCARD. Insert SD CARD. Power up the <%= TYPE_NAME %>.'
+IMX8M_VAR_DART_POST_FLASH = 'Set the SW7 BOOT SELECT switch to eMMC.'
```

Comment:
I think the pcb has either `ext` or `int` on this boot switch?

Reply:
Correct, solved